.husky/pre-commit

```diff
@@ -0,0 +1,26 @@
+#
```

Comment:
this file looks like only applicable to the frontend project, why put it in the top-level folder?

Reply:
I have modified it to the web folder.